index.html

```diff
@@ -4709,6 +4708,18 @@ <h2 id="AudioBufferSourceNode">
           This node has no <a>tail-time</a> reference.
         </p>
         <p>
+          A <dfn>playhead position</dfn> for an <a>AudioBufferSourceNode</a> is
```

Comment:
Actually I moved it here because it's now referenced in the definitions for `loopStart`, `loopEnd` and `offset` (which allowed me to remove some clumsy language for those parameters that you had commented on). Make sense?